pandas/core/generic.py

```diff
@@ -11586,7 +11586,7 @@ def tz_localize(
 
                 You can already get the future behavior and improvements through
                 enabling copy on write ``pd.options.mode.copy_on_write = True``
-        ambiguous : 'infer', bool-ndarray, 'NaT', default 'raise'
+        ambiguous : 'infer', bool, 'NaT', default 'raise'
```

Comment:
```suggestion
        ambiguous : 'infer', bool, bool-ndarray, 'NaT', default 'raise'
```

You can reference an example below where an array of bools can be applied